Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -296,6 +297,74 @@ _disableAffix         | bool    | Indicate whether to show the affix bar on the
 _disableContribution  | bool    | Indicate whether to show the `View Source` and `Improve this Doc` buttons.
 _gitContribute        | object  | Customize the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `path` to specify the folder for new overwrite files .
 
+#### 3.2.3 Separated metadata files for global metadata and file metadata
+
+Take global metadata for example, you can set `globalMetadataFiles` in `docfx.json` or `--globalMetadataFiles` in build command line. The usage of `fileMetadataFiles` is the same as `globalMetadataFiles`.
+
+There're some metadata file examples:
+
++ globalMetadata file example 
+    ```json
+    {
+        "_appTitle": "DocFX website",
+        "_enableSearch": "true"
+    }
+    ```
+
++ fileMetadata file example
+    ```json
+    {
+        "priority": {
+            "**.md": 2.5,
+            "spec/**.md": 3
+        },
+        "keywords": {
+            "obj/docfx/**": ["API", "Reference"],
+            "spec/**.md": ["Spec", "Conceptual"]
+        }
+    }
+    ```
+
+There're some examples about how to use separated metadata files.
+
++ use `globalMetadataFiles` in `docfx.json`
+    ```json
+    ...
+    "globalMetadataFiles": ["global1.json", "global2.json"],
+    ...
+    ```
+
++ use `--globalMetadataFiles` in build command line
+    ```
+    docfx build --globalMetadataFiles global1.json,global2.json
+    ```
+
++ use `fileMetadataFiles` in `docfx.json`
+    ```json
+    ...
+    "fileMetadataFiles": ["file1.json", "file2.json"],
+    ...
+    ```
+
++ use `--fileMetadataFiles` in build command line
+    ```
+    docfx build --fileMetadataFiles file1.json,file2.json
+    ```
+
+Note that, the metadata set in command line will merge with the metadata set in `docfx.json`. 
+
++ If the same key for global metadata was set, the order to be overriden would be: 
+    1. global metadata from docfx config file
+    2. global metadata from global metadata files
+    3. global metadata from command line
+    4. metadata from YAML header in current file
+
++ If the same file pattern for file metadata was set, the order to be overriden would be: 
+    1. file metadata from docfx config file
+    2. file metadata from file metadata files
+    3. metadata from YAML header in current file
+
+Given mulitple metadata files, the metadata in next file will override current metadata when the same key is set. In above example, the value set in `global2.json` will override the value set in `global1.json` if the same key exists.
```

Comment:
>next file [](start = 47, length = 9)

I think there is no clear definition about "next" file. For multiple files matching the same glob pattern, do we have an order for them? How about we just say the behavior is *undetermined*?